configure

```diff
@@ -4248,7 +4248,15 @@ then :
 
 fi
 
-# overlayfs features temporarily disabled pending fixes
+# The --overlay and related options are not supported anymore due to security
+# concerns (see CVE-2021-26910 and #4178), issues on newer kernels (see #2799)
+# and overall lack of interest in fixing and maintaining them.
+#
+# Support is unlikely to be restored, as it would require auditing the
+# overlayfs code to ensure that it is secure and that it works properly.
```

Comment:
Do we want "unlikely to be restored" or "disable until someone picks up the work".

Reply:
> Do we want "unlikely to be restored" or "disable until someone picks up the
> work".

Misc: I used stronger language to avoid giving false hope (as with the current
"temporarily disabled") that could lead to more new issues asking to restore
overlayfs, but I agree that it's good to note that it's still possibility with
enough work.

Updated with a bit of both; thoughts?

Reply:
Sounds good. Gives users no false expectation but underlines the reason (open-source development in free time).